tornado/autoreload.py

```diff
@@ -209,7 +209,10 @@ def _reload():
     global _reload_attempted
     _reload_attempted = True
     for fn in _reload_hooks:
-        fn()
+        try:
+            fn()
+        except Exception as e:
+            gen_log.warning("Error occured on reload: %s" % e)
```

Comment:
I see! Just committed fix with a wrapper function, should be better now. And I removed this try...except part.